account-kit/core/src/types.ts

```diff
@@ -107,6 +113,7 @@ type CreateStorageFn = (config?: {
 
 export type CreateConfigProps = RpcConnectionConfig & {
   sessionConfig?: AlchemySignerParams["sessionConfig"] & { domain?: string };
+  createSigner?: (config: ClientStoreConfig) => AlchemySigner;
```

Comment:
I wonder if we should put this within an `_internal` property of `CreateConfigProps` 🤔